ldscript.txt

```diff
@@ -34,7 +34,7 @@ SECTIONS
         build/sa2/src/main.o(.text);
         build/sa2/src/task.o(.text);
         build/sa2/src/sprite_1.o(.text);
-        build/sa2/asm/sprite.o(.text);
+        build/sa2/src/bg_update.o(.text);
```

Comment:
I think maybe we'll end up following katam here and naming `bg.c`, it's fine for now